env.sh

```diff
@@ -4,21 +4,25 @@
 
 # OS
 if [[ "$(uname)" == 'Darwin' ]]; then
-    readonly SYSTEM_NET='bridge'
-    SOURCE_DIR="$(dirname "$(realpath "$0")")"
+    readonly SOURCE_DIR="$(dirname "$(realpath "$0")")"
 else
-    readonly SYSTEM_NET="host"
-    SOURCE_DIR="$(dirname "$(readlink -f "$0")")"
+    readonly SOURCE_DIR="$(dirname "$(readlink -f "$0")")"
 fi
 
 readonly CONTAINER_NAME_HASH=$(echo "${SOURCE_DIR}" | md5sum | cut -d " " -f 1)
 if test -f "${SOURCE_DIR}/Cargo.toml"; then
+    readonly SYSTEM_NET='bridge'
     readonly CONTAINER_NAME="cita_build${CONTAINER_NAME_HASH}"
     readonly DOCKER_IMAGE='cita/cita-build:ubuntu-18.04-20191128'
 else
+    if [[ "$(uname)" == 'Darwin' ]]; then
+        readonly SYSTEM_NET='bridge'
+        EXPOSE='1337:1337'
+    else
+        readonly SYSTEM_NET="host"
```

Comment:
No need to specify `EXPOSE='1337:1337'` in unix system?

Reply:
Yes, in unix system network is host.

Reply:
okay, it works!